src/Chapter04/Listing04.28.OutOfScope.cs

```diff
@@ -27,11 +27,15 @@ public static void Main(string[] args)
             {
                 // ...
             }
+            #endregion INCLUDE
+            #if INCLUDE
+            #region INCLUDE
             #region HIGHLIGHT
             // ERROR: message is not in scope:
-            // System.Console.WriteLine(message);
+            System.Console.WriteLine(message);
```

Comment:
```suggestion
            Console.WriteLine(message);
```
To be consistent since it seems like we are stripping this elsewhere